website/src/components/FlaggableElement.tsx

```diff
@@ -184,40 +189,40 @@ interface textFlagLabels {
 const TEXT_LABEL_FLAGS: textFlagLabels[] = [
   // For the time being this list is configured on the FE.
   // In the future it may be provided by the API.
+  // {
+  //   attributeName: "fails_task",
+  //   labelText: "Fails to follow the correct instruction / task",
+  //   additionalExplanation: "__TODO__",
+  // },
+  // {
+  //   attributeName: "not_customer_assistant_appropriate",
+  //   labelText: "Inappropriate for customer assistant",
+  //   additionalExplanation: "__TODO__",
+  // },
   {
-    attributeName: "fails_task",
-    labelText: "Fails to follow the correct instruction / task",
-    additionalExplanation: "__TODO__",
-  },
-  {
-    attributeName: "not_customer_assistant_appropriate",
-    labelText: "Inappropriate for customer assistant",
-    additionalExplanation: "__TODO__",
-  },
-  {
-    attributeName: "contains_sexual_content",
+    attributeName: "sexual_content",
     labelText: "Contains sexual content",
   },
   {
-    attributeName: "contains_violent_content",
+    attributeName: "violence",
     labelText: "Contains violent content",
   },
-  {
-    attributeName: "encourages_violence",
-    labelText: "Encourages or fails to discourage violence/abuse/terrorism/self-harm",
-  },
-  {
-    attributeName: "denigrates_a_protected_class",
-    labelText: "Denigrates a protected class",
-  },
-  {
-    attributeName: "gives_harmful_advice",
-    labelText: "Fails to follow the correct instruction / task",
-    additionalExplanation:
-      "The advice given in the output is harmful or counter-productive. This may be in addition to, but is distinct from the question about encouraging violence/abuse/terrorism/self-harm.",
-  },
-  {
-    attributeName: "expresses_moral_judgement",
-    labelText: "Expresses moral judgement",
-  },
+  // {
+  //   attributeName: "encourages_violence",
+  //   labelText: "Encourages or fails to discourage violence/abuse/terrorism/self-harm",
+  // },
+  // {
+  //   attributeName: "denigrates_a_protected_class",
+  //   labelText: "Denigrates a protected class",
+  // },
+  // {
+  //   attributeName: "gives_harmful_advice",
+  //   labelText: "Fails to follow the correct instruction / task",
+  //   additionalExplanation:
+  //     "The advice given in the output is harmful or counter-productive. This may be in addition to, but is distinct from the question about encouraging violence/abuse/terrorism/self-harm.",
+  // },
+  // {
+  //   attributeName: "expresses_moral_judgement",
+  //   labelText: "Expresses moral judgement",
+  // },
```

Comment:
I guess the backend returns labels that are not part of this array?

Reply:
[I thought of fixing it as part of #493](https://github.com/LAION-AI/Open-Assistant/issues/493)
is it ok ?